Source/Krypton Components/Krypton.Toolkit/Controls Visuals/VisualMessageBoxForm.cs

```diff
@@ -954,6 +956,47 @@ private void UpdateHelp()
             }
         }
 
+        /// <summary>
+        /// When the user clicks the Help button, the Help file specified in the helpFilePath parameter
+        /// is opened and the Help keyword topic identified by the keyword parameter is Displayed.
+        /// The form that owns the message box (or the active form) also receives the HelpRequested event.
+        /// </summary>
+        private void LaunchHelp(IWin32Window? owner)
```

Comment:
Is there a test for this? There should already be a [Request] to have this function ality somewhere.

Reply:
Should be in the 'new' constructor.